frontend/src/lib/components/BridgePage/BridgePage.tsx

```diff
@@ -78,16 +73,15 @@ export function BridgePage({
     return (
         <div className={clsx('BridgePage', fixedWidth && 'BridgePage--fixed-width', className)}>
             <div className="BridgePage__main">
-                {!noHedgehog ? (
-                    <div className="BridgePage__art-wrapper">
-                        <div className="BridgePage__art">
+                {!noHedgehog || view === 'signup' ? (
+                    <div className="BridgePage__left-wrapper">
+                        <div className="BridgePage__left">
                             {!noLogo && sideLogo && (
                                 <div className="BridgePage__header-logo mb-4">
                                     <WelcomeLogo view={view} />
                                 </div>
                             )}
-                            {featureFlags[FEATURE_FLAGS.SIGNUP_PRODUCT_BENEFITS_EXPERIMENT] === 'test' &&
-                            showSignupCta ? (
+                            {view === 'signup' ? (
```

Comment:
I'm not a big fan of having conditionals like this, can't we split this component into two different components? One for the signup view, one, for anything else (basically the one we already had)?

Can be done in a follow up PR if you want, but I think it would make things cleaner